api_tests/lib/ledgers/ledger_runner.ts

```diff
@@ -90,13 +89,13 @@ export class LedgerRunner {
             if (ledger === "ethereum") {
                 const ethereumConfig = await this.getEthereumNodeConfig();
                 const erc20Wallet = new EthereumWallet(ethereumConfig);
-                global.tokenContract = await erc20Wallet.deployErc20TokenContract(
-                    global.projectRoot
+                this.global.tokenContract = await erc20Wallet.deployErc20TokenContract(
```

Comment:
Modifying your arguments is a difficult to understand design, would prefer return values here.